tonic/src/transport/server/mod.rs

```diff
@@ -720,6 +704,7 @@ impl<L> Server<L> {
                 .keep_alive_timeout(http2_keepalive_timeout)
                 .adaptive_window(http2_adaptive_window.unwrap_or_default())
                 .max_pending_accept_reset_streams(http2_max_pending_accept_reset_streams)
+                .max_local_error_reset_streams(http2_max_local_error_reset_streams)
```

Comment:
The required `hyper-util` version seems to be needed to be updated to `0.1.11`, which is the version this API was added.

https://github.com/hyperium/hyper-util/releases/tag/v0.1.11